doc/MANGO.tex

```diff
@@ -162,6 +162,8 @@ \subsection{Role within the VO Architecture}
 \section{Representing observed astronomical objects : Use Cases and  Requirements}
 
 \subsection{Use Cases}
+But this is not sufficient for tables with more complex cases based on multi-parameter cases.
+
```

Comment:
The reconstruction of  multi-parameter quantities if an explict requirement (section 2.2) that solves science use-cases. We can add a few words in the head of the use-cases section but not too much.

Reply:
line 165 not relevant here . may be an error in cut and paste